pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -759,18 +758,18 @@ public void unloadSLANamespace() throws Exception {
         PulsarAdmin adminClient = null;
         String namespaceName = getSLAMonitorNamespace(host, config);
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("Trying to unload SLA namespace {}", namespaceName);
-        }
+        LOG.info("Trying to unload SLA namespace {}", namespaceName);
 
         NamespaceBundle nsFullBundle = getFullBundle(new NamespaceName(namespaceName));
         if (!getOwner(nsFullBundle).isPresent()) {
             // No one owns the namespace so no point trying to unload it
+            // Next lookup will assign the bundle to this broker.
+            LOG.info("No one owns SLA namespace {}. Aborting unload", namespaceName);
```

Comment:
This would be the regular case. I'd like not to have this message each time the broker starts